src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java

```diff
@@ -363,5 +369,19 @@ private void enrichCaseDataWithJurisdictionCodes(CaseData caseData) {
         List<JurCodesTypeItem> jurCodesTypeItems = jurisdictionCodesMapper.mapToJurCodes(caseData);
         caseData.setJurCodesCollection(jurCodesTypeItems);
     }
+
+    private DocumentTypeItem createDocumentTypeItem(String typeOfDocument, UploadedDocumentType uploadedDoc) {
```

Comment:
maybe worth moving this method CaseDocumentService? would be more appropriate elsewhere I think